app/lib/frontend/handlers/pubapi.dart

```diff
@@ -262,11 +263,17 @@ class PubApi {
   // ****
 
   @EndPoint.get('/api/admin/users')
-  Future<AdminListUsersResponse> adminListUsers(Request request) async => null;
+  Future<AdminListUsersResponse> adminListUsers(Request request) async =>
+      adminBackend.listUsers(
+        continuationToken:
+            request.requestedUri.queryParameters['continuationToken'],
+      );
 
   @EndPoint.delete('/api/admin/users/<userId>')
-  Future<Response> adminRemoveUser(Request request, String userId) async =>
-      Response.notFound('no such thing');
+  Future<Response> adminRemoveUser(Request request, String userId) async {
+    await adminBackend.removeUser(userId);
+    return jsonResponse({'status': 'OK'});
```

Comment:
nit. `204` with no body is customary..

Reply:
I prefer to also have a string output, it is better for debugging (no contemplating whether the body got lost somewhere, is empty on purpose, or had other issues).